.github/vale/dicts/aiven.dic

```diff
@@ -17,6 +17,7 @@ codecs
 Conduktor
 connect
 Couchbase
+cqlsh
```

Comment:
I believe we are trying to avoid adding tool names in the vocab. I suggested few changes of tool_name to `tool_name` that should make this line not needed